docs/data-model.html

```diff
@@ -0,0 +1,868 @@
+        "kty": "OKP",
+        "crv": "X25519",
+        "x": "EXXinkMxdA4zGmwpOOpbCXt6Ts6CwyXyEKI3jfHkS3k"
+      }
+    }
+  ],
+  "authentication": [
+    "#z6Mks8mvCnVx4HQcoq7ZwvpTbMnoRGudHSiEpXhMf6VW8XMg"
+  ],
+  "assertionMethod": [
+    "#z6Mks8mvCnVx4HQcoq7ZwvpTbMnoRGudHSiEpXhMf6VW8XMg"
+  ],
+  "capabilityInvocation": [
+    "#z6Mks8mvCnVx4HQcoq7ZwvpTbMnoRGudHSiEpXhMf6VW8XMg"
+  ],
+  "capabilityDelegation": [
+    "#z6Mks8mvCnVx4HQcoq7ZwvpTbMnoRGudHSiEpXhMf6VW8XMg"
+  ],
+  "keyAgreement": [
+    "#z6LScrLMVd9jvbphPeQkGffSeB99EWSYqAnMg8rGiHCgz5ha"
+  ]
+},
+"didDocumentMetadata": {
+  "content-type": "application/did+json"
+},
+"didResolutionMetadata": {}
+}
+
+</pre
+      >
+    </section>
+
+    <section>
+      <h3 id="Currency"><dfn>Currency</dfn></h3>
+      <p>An amount of currency controlled by one or more <a>Key</a>s.</p>
+
+      <p>Here is an example of some bitcoin stored on a testnet.</p>
+
+      <pre class="example highlight" title="A bitcoin currency example">
+{
+"@context": ["https://w3id.org/wallet/v1"],
+"id": "https://live.blockcypher.com/btc-testnet/address/mu6vftfsBxJDhnUQVrZfwjzj5BL5WRvLUH/",
+"type": "Currency",
+"name": "BTC for Testing",
+"image": "https://via.placeholder.com/150",
+"description" : "Bitcoin reserved for testing",
+"tags": ["personal"],
+"amount": "7.00749119",
+"currency": "BTC",
+"controller": "did:example:123",
+"correlation": ["4058a72a-9523-11ea-bb37-0242ac130002"]
+}
+</pre
+      >
+
+      <p>Here is an example of some ethereum stored on a testnet.</p>
+
+      <pre class="example highlight" title="A currency example">
+{
+"@context": ["https://w3id.org/wallet/v1"],
+"id": "https://ropsten.etherscan.io/address/0x74fd35392f1c2a97c6080e0394b1995b0e63c8bf",
+"type": "Currency",
+"name": "ETH for Tips",
+"image": "https://via.placeholder.com/150",
+"description" : "Ethereum reserved for tipping software developers.",
+"tags": ["personal"],
+"amount": "8.142406194939252896",
+"currency": "ETH",
+"controller": "did:ethr:0x74fd35392f1c2a97c6080e0394b1995b0e63c8bf",
+"correlation": ["4058a72a-9523-11ea-bb37-0242ac130002"]
+}
+</pre
+      >
+
+      <p>
+        Here is an example of some ethereum stored on a testnet requiring
+        multisig.
+      </p>
+
+      <pre class="example highlight" title="A currency example">
+{
+"id": "https://ropsten.etherscan.io/address/0x658e4fe24b34589492b18b1a45294be0601606a9",
+"type": "Currency",
+"name": "ETH for Games",
+"image": "https://via.placeholder.com/150",
+"description" : "Ethereum reserved for games.",
+"tags": ["personal"],
+"amount": "6,271.91386730845876097",
+"currency": "ETH",
+"controller": "did:ethr:0x658e4fe24b34589492b18b1a45294be0601606a9",
+"quorum": 2,
+"correlation": ["4058a72a-9523-11ea-bb37-0242ac130002"]
+}
+</pre
+      >
+    </section>
+
+    <section>
+      <h3 id="Credential"><dfn>Credential</dfn></h3>
+
+      <p>A set of claims about a subject.</p>
+
+      <p>MAY be presented with one or more <a>Key</a>s.</p>
```

Comment:
Respec can/should be configured to handle plural keywords, which should not be broken by the `</a>` tag.
```suggestion
      <p>MAY be presented with one or more <a>Keys</a>.</p>
```